city_scrapers/spiders/chi_ssa_61.py

```diff
@@ -0,0 +1,74 @@
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa61Spider(CityScrapersSpider):
+    name = "chi_ssa_61"
+    agency = "Chicago Special Service Area #60 Hyde Park"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.downtownhydeparkchicago.com"]
+    start_urls = ["http://www.downtownhydeparkchicago.com/about/"]
+    location = {
+        "name": "Polsky Center for Innovation",
+        "address": "1452 East 53rd Street, 2nd Floor, Chicago, IL 60615"
+    }
+
+    def parse(self, response):
+        """Parse meetings on upcoming and minutes pages"""
+        self._validate_location(response)
+        today = datetime.now().replace(hour=0, minute=0)
+        for item in response.xpath('//div[@rel="1"]/ul/li/strong'):
+            text = item.xpath('./text()').extract_first()
+            if not re.search(r"[ pam\.]{2,5}", text):
+                continue
+            start = self._parse_start(text)
+            if not start or (start < today):
```

Comment:
Sorry,  I missed this on the first review, but we should also be scraping meetings in the past. They aren't currently posting minutes, but it looks like they [have in the past](https://www.downtownhydeparkchicagoinfo.com/minutes) and we'll want to be able to scrape those and attach them to meetings if they're available

Reply:
Okay, should we scrape the old agendas from this page as well? [Agendas from old website](https://www.downtownhydeparkchicagoinfo.com/agendas)

Reply:
I don't think we need to, since they aren't posted for 2019 yet

Reply:
the old minutes are on the old website too, should i still scrape for minutes?  

I can't find any recent minutes or agendas on the new website

Reply:
We can ignore them both for now since they haven’t been posted this year and might not be going forward

Reply:
sounds good